src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -557,7 +593,14 @@ LRESULT FancyZones::WndProc(HWND window, UINT message, WPARAM wparam, LPARAM lpa
 
     default:
     {

```

Comment:
Why this big `default` block exists if we also `if` for `message` in it? Can make this as a single switch-case?

Reply:
Yes, because these messages do not fulfil _Integer Constant Expression_, so they can't be labels in switch-case.